sklearn/ensemble/_forest.py

```diff
@@ -1570,9 +1594,10 @@ class RandomForestRegressor(ForestRegressor):
         Whether bootstrap samples are used when building trees. If False, the
         whole dataset is used to build each tree.
 
-    oob_score : bool, default=False
+    oob_score : bool or callable, default=False
         Whether to use out-of-bag samples to estimate the generalization score.
-        Only available if bootstrap=True.
+        By default the R2 score is used. Provide a callable to use a custom
+        scoring function. Only available if bootstrap=True.
```

Comment:
Can we update the docstrings for the other `oob_score` parameters matching the suggestion in https://github.com/scikit-learn/scikit-learn/pull/25177#discussion_r1052442359?